udata/core/user/activities.py

```diff
@@ -70,3 +87,17 @@ def write_activity_on_follow(follow, **kwargs):
             write_activity.delay(
                 UserFollowedUser, current_user._get_current_object(),
                 follow.following)
+
+
+@on_new_discussion.connect
+@on_new_discussion_comment.connect
+def write_activity_on_discuss(discussion, **kwargs):
+    if current_user.is_authenticated:
```

Comment:
Wow, that's scary. I updated that everywhere else and given the number of places I'm surprised we didn't hit any bug related to that!